include/silo/storage/column/date_column.h

```diff
@@ -34,6 +34,8 @@ class DateColumnPartition {
 
    void insert(const silo::common::Date& value);
 
+   void insertNull();
```

Comment:
This is unused. Is this expected? Or is `ColumnPartitionGroup::fill` wrong?

Reply:
Same for `InsertionColumnPartition<SymbolType>::insertNull()`, `IntColumnPartition::insertNull()` and `FloatColumnPartition::insertNull()`.